app.js

```diff
@@ -3,11 +3,14 @@ const app = express();
 const {
   getTopics
 } = require('./controllers/topics-controllers.js');
+const { getAPI } = require('./controllers/api-controllers.js')
+const { getArticleById } = require('./controllers/articles-controllers.js')
 
 
 
 app.get('/api/topics', getTopics);
-
+app.get('/api', getAPI)
+app.get('/api/articles/:article_id',getArticleById)
 
```

Comment:
There doesn't appear to be any error handler in your app so any errors, like 404 and 400 for the single article, won't be handled